force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -227,19 +273,24 @@ public with sharing class ClassesObjectsMethods {
      */
     public class Book {
         // Declare three public instance variables - title, author, pages
-        // title
+        // title 
         // author
         // pages
+        public String title;
+        public String author;
+        public Integer pages;
     
         // Book constructor to initialize the title and author
-        public Book() {
-            // Initialize the title and author
+        public Book(String title, String author) {
+           this.title = title;
+           this.author = author;// Initialize the title and author
         }
     
         // Method that returns the details of the book
         public String getBookDetails() {
+            String details = 'Title: ' + title + ', Author: ' + author;
```

Comment:
To be consistent, it's best to use `this` when referencing instance variables in your code:
```suggestion
            String details = 'Title: ' + this.title + ', Author: ' + this.author;
```